python/tank_vendor/ruamel_yaml/comments.py

```diff
@@ -12,7 +12,7 @@
 a separate base
 """
 
-from collections import MutableSet
+from collections.abc import MutableSet
```

Comment:
Should we update the ruamel_yaml package instead? 
The problem is we have a few local changes to this version so this would involve more test and time.

Reply:
I created the missing requirement file in a different branch - SG-31907.